Normalizer/DefaultMessageNormalizer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Bernard\BernardBundle\Normalizer;
+
+use Assert\Assertion;
+use Bernard\Message\DefaultMessage;
+use Bernard\Normalizer\PlainMessageNormalizer;
+
+class DefaultMessageNormalizer extends PlainMessageNormalizer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function normalize($object, $format = null, array $context = [])
+    {
+        @trigger_error('The '.__NAMESPACE__.'\DefaultMessageNormalizer class is deprecated and will removed in version 3.0. Use '.__NAMESPACE__.'\PlainMessageNormalizer instead.', E_USER_DEPRECATED);
```

Comment:
You sure `__NAMESPACE__` is correct here before PlainMessageNormalizer? I would rather use `PlainMessageNormalizer::class`